go.mod

```diff
@@ -18,17 +18,17 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/vmware/govmomi v0.22.2
-	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
+	golang.org/x/net v0.0.0-20210825183410-e898025ed96a
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/warnings.v0 v0.1.2 // indirect
-	k8s.io/api v0.22.1
-	k8s.io/apimachinery v0.22.1
+	k8s.io/api v0.23.0
+	k8s.io/apimachinery v0.23.0
 	k8s.io/apiserver v0.22.0
 	k8s.io/client-go v0.22.1
-	k8s.io/code-generator v0.22.1
-	k8s.io/klog/v2 v2.9.0
+	k8s.io/code-generator v0.23.0
+	k8s.io/klog/v2 v2.30.0
 	k8s.io/kubectl v0.22.0
```

Comment:
All the K8s versions need to match else you'll have issues with deps, please update them all to v0.23.0

Reply:
Will update. Thanks